packages/app-project/src/screens/ClassifyPage/components/YourStats/components/DailyClassificationsChart/DailyClassificationsChartContainer.js

```diff
@@ -1,56 +1,44 @@
 import counterpart from 'counterpart'
-import { inject, observer } from 'mobx-react'
 import { array, number, shape, string } from 'prop-types'
-import { Component } from 'react'
 
 import DailyClassificationsChart from './DailyClassificationsChart'
 
-function storeMapper (stores) {
-  const { project, user: { personalization: { counts, stats: { thisWeek } } } } = stores.store
-  return {
-    counts,
-    thisWeek,
-    projectName: project['display_name']
-  }
+const defaultCounts = {
+  today: 0
 }
 
-@inject(storeMapper)
-@observer
-class DailyClassificationsChartContainer extends Component {
-  render () {
-    const TODAY = new Date()
-    const { counts, projectName, thisWeek } = this.props
-    const stats = thisWeek.map(stat => {
-      const day = new Date(stat.period)
-      const locale = counterpart.getLocale()
-      const count = (day.getDay() === TODAY.getDay()) ? counts.today : stat.count
-      const longLabel = day.toLocaleDateString(locale, { weekday: 'long' })
-      const alt = `${longLabel}: ${count}`
-      const label = day.toLocaleDateString(locale, { weekday: 'narrow' })
-      return Object.assign({}, stat, { alt, count, label, longLabel })
-    })
-    return (
-      <DailyClassificationsChart
-        stats={stats}
-        projectName={projectName}
-      />
-    )
-  }
+function DailyClassificationsChartContainer({
+  counts = defaultCounts,
+  projectName,
+  thisWeek = []
+}) {
+  const TODAY = new Date()
+  const stats = thisWeek.map(stat => {
+    const day = new Date(stat.period)
+    const locale = counterpart.getLocale()
+    const count = (day.getUTCDay() === TODAY.getUTCDay()) ? counts.today : stat.count
```

Comment:
I'm not sure if changing both of these to `getUTCDay()` is correct. If the stats server sends back daily stats stamped with `2021-06-14T00:00:00Z`, and the local clock is CDT eg. `2021-06-14T20:00:00-05:00`, should we compare the UTC day (from the server) with the local day? Otherwise, I think the local UTC day will be one day ahead of the day in the server response.
```js
    const count = (day.getUTCDay() === TODAY.getDay()) ? counts.today : stat.count
```

Reply:
So in that example, at 8pm on Monday, any classifications you made after midnight in London would be displayed as Tuesday. I think.

Reply:
I've verified that this is still a bug by changing my mac's clock today (Wednesday) to a time zone that's still on Tuesday. Any classifications I made were shown as Wednesday, one day ahead in my local future (cue spooky Twilight Zone music.) Fixed by making the change I outlined in the comments: use UTC to get a day number from the stats API data, but the local clock to get the local day number.